### PR TITLE
update-view

### PR DESCRIPTION
**Pull request by tu01titon0**

### CODE REVIEW

File: app/views/testview/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom shadow-sm">
+<h5 class="my-0 mr-md-auto font-weight-normal">Company name</h5>
+<nav class="my-2 my-md-0 mr-md-3">
+  <a class="p-2 text-dark" href="#">Features</a>
+  <a class="p-2 text-dark" href="#">Enterprise</a>
+  <a class="p-2 text-dark" href="#">Support</a>
+  <a class="p-2 text-dark" href="#">Pricing</a>
```
Comment (max-mfv) [inline app/views/testview/_header.html.erb:7]: - Change user, pet, listing path
- using `link_to`

File: app/views/testview/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<footer class="pt-4 my-md-5 pt-md-5 border-top">
```
Comment (max-mfv) [inline app/views/testview/_footer.html.erb:1]: rename folder testview -> shared

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,6 +10,10 @@
   </head>
 
   <body>
+    <div class="container">
+    <%= render "testview/header" %>
     <%= yield %>
+    <%= render "testview/footer" %>
+    </div>
```
Comment (max-mfv) [inline app/views/layouts/application.html.erb:17]: header and footer should be outside of the container
```
  <%= render "testview/header" %>
  <div class="container">
     <%= yield %>
  </div>
  <%= render "testview/footer" %>
```

File: app/views/testview/_header.html.erb (hunk 1 above)
Comment (max-mfv) [reply app/views/testview/_header.html.erb:7]: Menu: Users | Pets | Listings

Comment (max-mfv) [reply app/views/testview/_header.html.erb:7]: Company name --> Pet Hotel

File: app/views/testview/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+</nav>
+<a class="btn btn-outline-primary" href="#">Sign up</a>
```
Comment (max-mfv) [inline app/views/testview/_header.html.erb:9]: Remove **Sign Up**

File: app/views/shared/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom shadow-sm">
+<h5 class="my-0 mr-md-auto font-weight-normal">Pet Hotel</h5>
+<nav class="my-2 my-md-0 mr-md-3">
+  <%= link_to "User", "/users", class:"p-2 text-dark" %>
+  <%= link_to "Pets", "/pets", class:"p-2 text-dark" %>
+  <%= link_to "Listing", "/listings", class:"p-2 text-dark" %>
```
Comment (toilapate) [inline app/views/shared/_header.html.erb:6]: Using path helper:
1. Check **rails routes**
2. Using:
"/users" --> user_path
"/pets" --> pets_path
"/listings" --> listings_path

**Pull request merged**